README.md

```diff
@@ -29,3 +29,73 @@ More information: <a href="http://rosin-project.eu">rosin-project.eu</a>
 This project has received funding from the European Union’s Horizon 2020  
 research and innovation programme under grant agreement no. 732287. 
 
+
+
+
+Welcome to the ROS End-Effector Action finder 
+
+## Warning
+From 28-01-2020 the use_gui param gives an error because it is deprecated. This causes the sliders of joint 
+state publisher not shown. To solve : 
+```bash
+sudo apt install ros-kinetic-joint-state-publisher-gui
+```
+
```

Comment:
Reminder for explicitly request moveit dependency

Reply:
Right! I added it in the last commit